UI/src/view/linear_asset/laneModellingLayer.js

```diff
@@ -319,6 +338,9 @@
           selectedAsset.startMeasure === asset.startMeasure && selectedAsset.endMeasure === asset.endMeasure; }) ;
       });
       me.vectorSource.addFeatures(style.renderFeatures(allButSelected));
+      var oneWaySignsDraft = me.getOneWaySignsForLanes(allButSelected);
```

Comment:
tän this ja me tarvettä tässä ihmettelen. me on vaan alias tolle this jotta ei tulisi nimi törmäyksiä ja ehkä vähän luettavampaa. käytä joko me tai this mut ei sekaisin. hämää tulevia lukioita